src/constants/index.ts

```diff
@@ -99,6 +99,19 @@ export const TRI: { [chainId in ChainId]: Token } = {
   )
 }
 
+export const XTRI: { [chainId in ChainId]: Token } = {
+  [ChainId.FUJI]: new Token(ChainId.FUJI, ZERO_ADDRESS, 18, 'xTRI', 'TriBar Token'),
+  [ChainId.AVALANCHE]: new Token(ChainId.AVALANCHE, ZERO_ADDRESS, 18, 'xTRI', 'TriBar Token'),
+  [ChainId.POLYGON]: new Token(ChainId.POLYGON, ZERO_ADDRESS, 18, 'xTRI', 'TriBar Token'),
+  [ChainId.AURORA]: new Token(
+    ChainId.AURORA,
+    '0x802119e4e253D5C19aA06A5d567C5a41596D6803',
```

Comment:
can we move this ro the sdk repo?
@0xdon

Reply:
Deployed - please import the latest version of the trisolaris sdk into package.json @specialist0x. It's version 1.6.0 - https://www.npmjs.com/package/@trisolaris/sdk. From the SDK you can import XTRI. And remove it from the constants file here.

Reply:
👌🏽

Reply:
Adding update based on offline chat:

- Attempted removing this, but wasn't able to import from `@trisolaris/sdk`
- Left this change in place to ensure stability